### DISCUSSION TITLE
0.85.0

### DISCUSSION THREAD

**Discussion by achantavy** [cartography-cncf/cartography#1246] [General]
## What's Changed
* Fix #1088: Refactor AWS Inspector to new model by @achantavy in https://github.com/lyft/cartography/pull/1241
* fix :#1218, #1152 - refactor EC2 network interface, private ip by @achantavy in https://github.com/lyft/cartography/pull/1219
* 0.85.0 by @achantavy in https://github.com/lyft/cartography/pull/1245

**Full Changelog**: https://github.com/lyft/cartography/compare/0.84.0...0.85.0

<hr /><em>This discussion was created from the release <a href='https://github.com/lyft/cartography/releases/tag/0.85.0'>0.85.0</a>.</em>

**Comment by jychp** [comment]
Information are available in [release note](https://github.com/cartography-cncf/cartography/releases/tag/0.85.0)
